scripts/dashboard/dashboard.sh

```diff
@@ -41,15 +41,15 @@ do
 done
 cat ${BASEDIR}/students.txt|while read STUDENT
```

Comment:
As you moved `students.txt` to dashboard/, can you fix `README.ME`, L276?

Reply:
done.